resources/js/features/events/components/AchievementSet/AchievementDateMeta/AchievementDateMeta.tsx

```diff
@@ -51,19 +51,19 @@ export const AchievementDateMeta: FC<AchievementDateMetaProps> = ({
       data-testid="date-meta"
       className={cn('gap-x-2 text-[0.63rem] text-neutral-400/70 light:text-neutral-500', className)}
     >
-      {isActive && activeUntil ? (
+      {isActive && activeThrough ? (
         <p className="text-green-400">
-          {t('Active until {{date}}', { date: formatDate(activeUntil, 'll') })}
+          {t('Active until {{date}}', { date: formatDate(activeThrough, 'll') })}
```

Comment:
This should say 'Active through {date}' per our discussion [here](https://github.com/RetroAchievements/RAWeb/pull/3024#discussion_r1907458742).

Reply:
Updated in latest.